snuba/state/cache/redis/backend.py

```diff
@@ -136,13 +139,15 @@ def build_notify_queue_key(task_ident: str) -> str:
 
         if timer is not None:
             timer.mark("cache_get")
+        metric_tags = timer.tags if timer is not None else {}
 
         # This updates the stats object and querylog
         record_cache_hit_type(result[0])
 
         if result[0] == RESULT_VALUE:
             # If we got a cache hit, this is easy -- we just return it.
             logger.debug("Immediately returning result from cache hit.")
+            metrics.increment("cache_hit", tags=metric_tags)
```

Comment:
is this going to cause an issue with sending too many messages to Datadog e.g. do we need some kind of sampling?

Reply:
We already track cache hits/misses etc., through the timer. Is there a reason this needs to be separate?

Reply:
I'll delete this one